apps/app/src/app/shared/layout/side-nav/side-nav.ts

```diff
@@ -17,7 +17,8 @@ import { HlmSidebarImports } from '@spartan-ng/helm/sidebar';
 			class="flex-1 [--sidebar-width:220px]! [--top-spacing:0] lg:[--sidebar-width:240px]! lg:[--top-spacing:calc(var(--spacing)*4)]"
 		>
 			<hlm-sidebar
-				sidebarContainerClass="text-sidebar-foreground sticky top-[calc(var(--header-height)+1px)] z-30 hidden h-[calc(100svh-var(--footer-height)+2rem-var(--stable-height))] w-(--sidebar-width) flex-col border-r-transparent bg-transparent lg:flex"
+				class="sticky top-[calc(var(--header-height)+1px)] hidden lg:flex"
+				sidebarContainerClass="text-sidebar-foreground sticky top-[calc(var(--header-height)+1px)] z-30  h-[calc(100svh-var(--footer-height)+2rem-var(--stable-height))] w-(--sidebar-width) flex-col border-r-transparent bg-transparent "
```

Comment:
@MerlinMoos please rebase to main as `hlm-sidebar` was updated, maybe both classes are not required anymore.